guides/doc-Configuring_Load_Balancer/master.adoc

```diff
@@ -4,6 +4,12 @@ include::common/header.adoc[]
 
 = {ConfiguringLoadBalancerDocTitle}
 
+// This guide only works on content proxies, which is Katello only
+ifdef::HideDocument,foreman-deb,foreman-el[]
```

Comment:
```suggestion
ifdef::HideDocument[]
```
Aren't those redundant when HideDocument is only defined in foreman-deb and foreman-el builds?

Reply:
Kind of. Perhaps it's a premature optimization for when the guide would be ready. Now it's just not usable at all. I guess we should keep it simple and deal with that stuff when we do get to it.